exercises/julia-1-a/README.md

```diff
@@ -0,0 +1,103 @@
+## Instructions
+
+In this exercise, you're going to implement a program that determines the state of a [tic-tac-toe](https://en.wikipedia.org/wiki/Tic-tac-toe) game.
+(_You may also know the game as "noughts and crosses" or "Xs and Os"._)
+
+The games is played on a 3x3 grid.
+Players take turns to place `X`s and `O`s on the grid.
+The game ends when one player has won by placing three of marks in a row, column, or along a diagonal of the grid, or when the entire grid is filled up.
+
+Without loss of generality we will assume that `X` starts.
```

Comment:
I don't know what this means. I'm also not sure it's needed for this part?

Reply:
It's needed because it decides which boards are invalid due to taking turns out of order. I can reduce it to

```suggestion
For the entire exercise, we will assume that `X` starts.
```

The w.l.o.g. means that we don't have to consider games where `O` starts because the solution/implementation would be nearly identical with the only exception that you'd first replace `X`->`O` and `O`->`X`.

Reply:
I also found the w.l.o.g. a bit hard to read. I like the suggested change!